6_visualize.yml

```diff
@@ -27,19 +27,18 @@ targets:
   # into an RDS file with a corresponding .rds.ind indicator file.
 
   # right here we will create and run a task table that will be responsible for generating
-  # the components, e.g., '6_visualize/out/style/storm_point_20170926_11.rds.ind' through
-  # '6_visualize/out/style/streamdata_20170926_11.rds.ind', one per column=step. and then the final
+  # the components, e.g., '5_vizprep/out/storm_point_20170926_11.rds.ind' through
+  # '5_vizprep/out/streamdata_20170926_11.rds.ind', one per column=step. and then the final
```

Comment:
i think you're proposing that the task table generated in 6_visualize puts some files into 5_vizprep. that seems like an unnecessary bending of our usual project structure practices; i'd rather keep these output files in 6_visualize if you can be persuaded

Reply:
I did this before looking at the task table PR, so I was envisioning a visualization prep stage that would precede this visualize stage. If we are going a different route, I think we can change this easily.

Reply:
OK, let's change it. You proposed a viz prep stage yesterday, so that does not surprise me, but then i had suggested on the phone (and want to keep pushing for this) that all of the dynamic files should live in a single task table, whether they're prep or actually create a png, because of the funkiness about having task-specific targets in one task file depend on task-specific targets in a different task file. So that's why my comment above (deleted lines 30 and 31) had them in a subfolder within phase 6.

Reply:
Yep, agreed. can change. I do remember that, but hadn't thought about how to make the connection w/ the other task table yet. Doing that now that I see the other work and your suggestions here.

Reply:
:+1: